rocAL/rocAL/include/loaders/audio/node_audio_loader_single_shard.h

```diff
@@ -0,0 +1,60 @@
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#pragma once
+#include "node.h"
+#include "audio_loader_sharded.h"
+#include "graph.h"
+
+class AudioLoaderSingleShardNode: public Node {
+public:
+
+    AudioLoaderSingleShardNode(rocalTensor *output, void* device_resources);
+    ~AudioLoaderSingleShardNode() override;
+    /// \param user_shard_count shard count from user
+    /// \param  user_shard_id shard id from user
+    /// \param source_path Defines the path that includes the Audio dataset
+    /// \param load_batch_count Defines the quantum count of the Audios to be loaded. It's usually equal to the user's batch size.
+    /// \param storage_type Determines the storage type
+    /// \param decoder_type Determines the decoder_type
+    /// \param shuffle Determines if the user wants to shuffle the dataset or not.
+    /// \param loop Determines if the user wants to indefinitely loops through images or not.
+    /// \param load_batch_count The batch size.
+    /// \param mem_type Memory type, host or device
+    /// \param meta_data_reader Determines the meta-data information
+    /// \param last_batch_policy Describes the last batch policy behavior when there are not enough samples in the epoch to fill a whole batch (FILL, DROP & PARTIAL).
+    /// \param last_batch_padded Determines whether the tail of the data consists of data from the next shard (False) or is duplicated dummy data (True).
+    /// \param stick_to_shard Determines after each epoch if the pipeline advances to the next shard to increase the entropy of the data that is seen by this pipeline or not.
+    /// \param shard_size Number of samples in the shard for the wrapped pipeline. Providing -1 means that the iterator will work until StopIteration is raised from the inside of iterator.
+    /// The loader will repeat Audios if necessary to be able to have Audios in multiples of the load_batch_count,
+    /// for example if there are 10 Audios in the dataset and load_batch_count is 3, the loader repeats 2 Audios as if there are 12 Audios available.
+    void init(unsigned shard_id, unsigned shard_count, const std::string &source_path, const std::string &source_file_list,
+              StorageType storage_type, DecoderType decoder_type, bool shuffle, bool loop,
+              size_t load_batch_count, RocalMemType mem_type, std::shared_ptr<MetaDataReader> meta_data_reader, RocalBatchPolicy _last_batch_policy = RocalBatchPolicy::BATCH_FILL, bool last_batch_padded = false, bool stick_to_shard = false, int shard_size=-1);
+    std::shared_ptr<LoaderModule> get_loader_module();
```

Comment:
Please align params in last row in init function, please add space before and after = operator

Reply:
Aligned

Reply:
Resolved